model/model.go

```diff
@@ -537,7 +537,18 @@ type ViewInfo struct {
 	Security    ViewSecurity       `json:"view_security"`
 	SelectStmt  string             `json:"view_select"`
 	CheckOption ViewCheckOption    `json:"view_checkoption"`
-	Cols        []CIStr            `json:"view_cols"`
+	// TODO: Remove this attribute when upgrade to next release version
+	Cols            []CIStr `json:"view_cols"`
+	SelectStmtQuote string  `json:view_select_quote`
```

Comment:
Seems we don't need to add a new field. We can traverse `SelectStmt` and replace ` to " to obtain the ansi quoted statement.

Reply:
@zz-jason Can we remove attr `Cols` when upgrade to 4.0?